src/utils/calc.rs

```diff
@@ -234,6 +234,16 @@ impl SlotCalculator {
         self.point_within_slot(chrono::Utc::now().timestamp() as u64)
     }
 
+    /// The current number of milliseconds into the slot.
+    pub fn current_point_within_slot_ms(&self) -> Option<u64> {
+        let now = chrono::Utc::now();
+        let timestamp = now.timestamp() as u64;
```

Comment:
https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.timestamp_millis

Reply:
rather than retrieving the timestamp from the chrono object 2 times you can retrieve it one time, and then operate on it. this is clearer and likely faste4